python/cugraph-pyg/cugraph_pyg/loader/link_neighbor_loader.py

```diff
@@ -130,25 +127,6 @@ def __init__(
         batch_size: int (optional, default=16)
             The number of input nodes per output minibatch.
             See torch.utils.dataloader.
-        directory: str (optional, default=None)
-            The directory where samples will be temporarily stored,
-            if spilling samples to disk.  If None, this loader
-            will perform buffered in-memory sampling.
-            If writing to disk, setting this argument
-            to a tempfile.TemporaryDirectory with a context
-            manager is a good option but depending on the filesystem,
-            you may want to choose an alternative location with fast I/O
-            intead.
-            See cugraph.gnn.DistSampleWriter.
-        batches_per_partition: int (optional, default=256)
-            The number of batches per partition if writing samples to
-            disk.  Manually tuning this parameter is not recommended
-            but reducing it may help conserve GPU memory.
-            See cugraph.gnn.DistSampleWriter.
-        format: str (optional, default='parquet')
-            If writing samples to disk, they will be written in this
-            file format.
-            See cugraph.gnn.DistSampleWriter.
```

Comment:
If you do something like this on this branch:

```shell
git grep -E 'cugraph\.gnn'
```

You'll see more docs references like "See cugraph.gnn.DistSampleWriter.".

Should those be updated to reference something like `cugraph_pyg.sampler.distributed_sampler.DistributedNeighborSampler`?

Reply:
Should be fixed now.